catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -168,7 +168,7 @@ public LibraryManager get_manager(String manager_type, AccessCriteria access_cri
                     poa.activate_object_with_id(managerId.getBytes(Charset.forName(NsiliEndpoint.ENCODING)),
                             catalogMgr);
                 } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-                    LOGGER.error("Error activating CatalogMgr: {}", e);
+                    LOGGER.info("Error activating CatalogMgr: {}", e);
```

Comment:
This is calling `toString()` on the exception instead of logging its stacktrace.